pkg/segment/aggregations/segaggs_test.go

```diff
@@ -2718,20 +2827,50 @@ func Test_PerformWindowStreamStatsOnSingleFunc(t *testing.T) {
 
 	windowSize := 3
 	expectedLen := []int{1, 2, 3, 3, 3, 3, 3, 3, 3, 3}
-	expectedValuesCount := []float64{1, 2, 3, 3, 3, 3, 3, 3, 3, 3}
-	values := []float64{7, 2, 5, 1, 6, 3, 8, 4, 9, 2}
-	expectedValuesSum := []float64{7, 9, 14, 8, 12, 10, 17, 15, 21, 15}
-	expectedValuesAvg := []float64{7, 4.5, 4.666666666666667, 2.6666666666666665, 4, 3.3333333333333335, 5.666666666666667, 5, 7, 5}
-	expectedValuesMax := []float64{7, 7, 7, 5, 6, 6, 8, 8, 9, 9}
 	expectedMaxLen := []int{1, 2, 2, 2, 1, 2, 1, 2, 1, 2}
-	expectedValuesMin := []float64{7, 2, 2, 1, 1, 1, 3, 3, 4, 2}
 	expectedMinLen := []int{1, 1, 2, 1, 2, 2, 2, 2, 2, 1}
 	expectedNilLen := []int{0, 0, 0, 0, 0, 0, 0, 0, 0, 0}
-	expectedValuesRange := []float64{0, 5, 5, 4, 5, 5, 5, 5, 5, 7}
-	expectedValuesCardinality := []float64{1, 2, 3, 3, 3, 3, 3, 3, 3, 3}
-	measureFunctions := []utils.AggregateFunctions{utils.Count, utils.Sum, utils.Avg, utils.Max, utils.Min, utils.Range, utils.Cardinality}
-	expectedFuncValues := [][]float64{expectedValuesCount, expectedValuesSum, expectedValuesAvg, expectedValuesMax, expectedValuesMin, expectedValuesRange, expectedValuesCardinality}
-	expectedFuncValues2 := [][]float64{expectedValuesCount, expectedValuesSum, expectedValuesSum, expectedValuesMax, expectedValuesMin, expectedValuesRange, expectedValuesCardinality}
+
+	expectedValuesCount := append(make([]interface{}, 0), 1, 2, 3, 3, 3, 3, 3, 3, 3, 3)
+	values := append(make([]interface{}, 0), 7, 2, 5, 1, 6, 3, 8, 4, 9, 2)
+	expectedValuesSum := append(make([]interface{}, 0), 7, 9, 14, 8, 12, 10, 17, 15, 21, 15)
```

Comment:
nit: it's a bit more common in Go and a little more concise to do this like:
```
expectedValuesSum := []interface{}{7, 9, 14, 8, 12, 10, 17, 15, 21, 15}
```

Reply:
addressed in the latest commit.